websites/A/AnimeDrive/metadata.json

```diff
@@ -0,0 +1,25 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"id": "291136400068444161",
+		"name": "ferivoq"
+	},
+	"service": "AnimeDrive",
+	"description": {
+		"en": "AnimeDrive is an anime streaming website with Hungarian subs/dubs"
+	},
+	"url": [
+		"animedrive.hu",
+		"www.animedrive.hu"
+	],
+	"version": "1.0.0",
+	"logo": "https://cdn.discordapp.com/icons/702098578251841577/be50b5dbcb30bd99512b6df29284db0c.webp",
```

Comment:
Logo should be 512x512px. (You may need to re-upload the asset to an image host, such as Imgur to avoid issues with query strings in this file)